test/al_aws_collector_test.js

```diff
@@ -1118,7 +1182,7 @@ describe('al_aws_collector error tests', function() {
     it('checkin error', function(done) {
         AlAwsCollector.load().then(function(creds) {
             var collector = new AlAwsCollector(
-                context, 'cwe', AlAwsCollector.IngestTypes.SECMSGS,'1.0.0', creds, undefined, [], []);
+                context, 'cwe', AlAwsCollector.IngestTypes.SECMSGS,'1.0.0', creds, undefined, [], [],[]);
```

Comment:
Shouldn't we leave this constructor as is to ensure that any dependent code doesn't need to be changed?

Reply:
sure .I will update.